apps/drupal-default/particle_theme/templates/form/views-exposed-form.html.twig

```diff
@@ -16,6 +16,6 @@
   #}
 {{ q }}
 {% endif %}
-<div class="form-inline clearfix mb-3 xl:w-1/3">
+<div class="form-inline clearfix">
```

Comment:
![Screen Shot 2021-03-08 at 2 24 15 PM](https://user-images.githubusercontent.com/39039024/110370679-fcdfdf80-8019-11eb-843c-1a07c6d9fdb4.png)
flow-root > clearfix
https://tailwindcss.com/docs/upgrading-to-v2
Idk if the change makes sense in this scenario but as a heads up.